app/build.gradle

```diff
@@ -1,306 +0,0 @@
-/*
- * Copyright (c) 2016-2018 Carmen Alvarez
- *
- * This file is part of Poet Assistant.
- *
- * Poet Assistant is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * Poet Assistant is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with Poet Assistant.  If not, see <http://www.gnu.org/licenses/>.
- */
-buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.github.ben-manes:gradle-versions-plugin:0.52.0'
-    }
-}
-
-plugins {
-    id 'jacoco'
-    id 'com.google.devtools.ksp'
-}
-apply plugin: 'com.android.application'
-apply plugin: 'com.github.ben-manes.versions'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-kapt'
-android {
-    compileSdkVersion 35
-
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
-    kotlinOptions {
-        jvmTarget = '1.8'
-    }
-
-    buildFeatures {
-        dataBinding = true
-        buildConfig = true
-    }
-    jacoco {
```

Comment:
This `jacoco` block has changed, it's now a `testCoverage` block. The contents are slightly different.